configs/sst_cs_plumbers-math-c9s.yaml

```diff
@@ -0,0 +1,17 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Math
+  description: Various math related libraries
```

Comment:
These need to be changed so as not to conflict with your existing Math workload.  Perhaps this should be renamed to ATLAS (both here and -atlas in the file name).